elastic/kibana
# [Repsonse Ops][Alerting] Error message hidden after closing action accordion

Resolves [#128041](https://github.com/elastic/kibana/issues/128041)

## Summary
- Added an error icon with a tooltip that will show up if you have errors in the action accordion and close the accordion
<img width="578" alt="Screen Shot 2022-07-19 at 3 48 22 PM" src="https://user-images.githubusercontent.com/109488926/179836115-9a0071db-4183-41a6-92f1-5951f08a603b.png">
- Fixed the alignment of the badge in the action accordion header

### Checklist

Delete any items that are not applicable to this PR.

### To Verify
- Verify that if you close action accordion before completing all the fields and click `Save` an error icon shows up indicating errors in the form
